www/js/app.js

```diff
@@ -426,9 +428,10 @@ define(['jquery', 'zimArchiveLoader', 'uiUtil', 'settingsStore','abstractFilesys
     });
     $('#btnAbout').on('click', function() {
         // Highlight the selected section in the navbar
-        $('#liHomeNav').attr("class","");
-        $('#liConfigureNav').attr("class","");
-        $('#liAboutNav').attr("class","active");
+        document.getElementById('liHomeNav').setAttribute("class", "");
+        document.getElementById('liConfigureNav').setAttribute("class", "");
+        document.getElementById('liAboutNav').setAttribute("class", "active");
```

Comment:
It looks like there's an opportunity to regularize the quotation mark style here -- could you change all the above to single quotation marks?